appinfo/info.xml

```diff
@@ -18,7 +18,7 @@
 	<screenshot small-thumbnail="https://raw.githubusercontent.com/ACTom/files_mindmap/master/screenshots/2-small.png">https://raw.githubusercontent.com/ACTom/files_mindmap/master/screenshots/2.png</screenshot>
 	<screenshot small-thumbnail="https://raw.githubusercontent.com/ACTom/files_mindmap/master/screenshots/3-small.png">https://raw.githubusercontent.com/ACTom/files_mindmap/master/screenshots/3.png</screenshot>
 	<dependencies>
-		<nextcloud min-version="14" max-version="24"/>
+		<nextcloud min-version="25" max-version="25"/>
```

Comment:
If the change you did in templates renders 0.0.27 incompatible to NC24, then this is correct. If it should work in older NC versions with your templates change, then min-version should be set to 22 here.

Reply:
I'm not very fluent with modern PHP. And PHP in general anyway. I tried to skim through nextcloud's repo and couldn't understand when OC_App showed up and when OCP\App was removed. I actually even found OCP\App in 25.0.0 so I have no idea why it doesn't work anymore here. Maybe there's actually a bug to fix in nextcloud? I also tried to understand if OC_App could work in NC 24 and below and huh... no idea either.

Could anyone try my PR with an older version of nextcloud please?

Reply:
I'm also not a programmer, and probably less insight in API changes than you have. But: If it works in NC24 and 25 for you, just set -min to 24 (or even 25) and -max to 25 as nothing else changed. Users with NC<24 will then get 0.0.26 and users with ≥25 will get 0.0.27 – which is fine for both of them. I see no point in finding the broadest possible solution. I suspect there would be much more to refactor once one started upon this app ;) – but this PR is only about getting status quo running easily on NC25, right? I'm very impressed that you found the culprit with OC_App anyway. I changed it in my instances and apart from the to-be-expected signature error, I can use mindmaps again, so 👍🏻

Reply:
I don't know if it works with 24. I only know it works with 25.

Reply:
Then I also think that it's best to leave -min and -max as you set them (both 25). Sorry for the noise.